src/discussions/comments/data/__factories__/comments.factory.js

```diff
@@ -51,3 +51,25 @@ Factory.define('commentsResult')
     const len = (pageSize * page <= count) ? pageSize : count % pageSize;
     return Factory.buildList('comment', len, { thread_id: threadId, parent_id: parentId });
   });
+
+Factory.define('commentsResponsesResult')
+  .option('count', null, 3)
+  .option('page', null, 1)
+  .option('pageSize', null, 5)
+  .option('threadId', null, 'test-thread')
+  .option('parentId', null, 'test-comment')
+  .attr('pagination', ['parentId', 'count', 'page', 'pageSize'], (parentId, count, page, pageSize) => {
+    const numPages = Math.ceil(count / pageSize);
+    const next = (page < numPages) ? `http://test.site/api/discussion/v1/comments/${parentId}/?page=${page + 1}` : null;
+    const prev = (page > 1) ? `http://test.site/api/discussion/v1/comments/${parentId}/?page=${page - 1}` : null;
+    return {
+      next,
+      prev,
+      count,
+      num_pages: numPages,
+    };
+  })
+  .attr('results', ['count', 'pageSize', 'page', 'parentId', 'threadId'], (count, pageSize, page, parentId, threadId) => {
+    const len = (pageSize * page <= count) ? pageSize : count % pageSize;
+    return Factory.buildList('comment', len, { thread_id: threadId, parent_id: parentId });
+  });
```

Comment:
I don't think you really need this, it's the same as "commentsResult" with the parent id set.

Reply:
Did see that at the time I was writing it, but you are right, thank you for spotting it.
I will remove it.